src/mmtk.rs

```diff
@@ -135,6 +135,9 @@ unsafe impl<VM: VMBinding> Send for MMTK<VM> {}
 impl<VM: VMBinding> MMTK<VM> {
     /// Create an MMTK instance. This is not public. Bindings should use [`MMTKBuilder::build`].
     pub(crate) fn new(options: Arc<Options>) -> Self {
+        // Set the static variable for mmap annotation.
+        crate::util::memory::MMAP_ANNOTATION.store(*options.mmap_annotation, Ordering::SeqCst);
```

Comment:
You don't need to be `SeqCst` here since there are no other threads right now (theoretically).

Reply:
In the end, I removed the option `options.mmap_annotation`.